model/clusters_mgmt/v1/upgrade_policy_state_resource.model

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages a specific upgrade policy state.
+resource UpgradePolicyState {
+	// Retrieves the details of the upgrade policy state.
+	method Get {
+		out Body UpgradePolicyState
+	}
+
+	// Update the upgrade policy state.
+	method Update {
```

Comment:
How will this be handled in terms of permissions? we don't want the user to be allowed to update the state, only the operator, right?

Reply:
Only the operator should access this endpoint, but due to the auth method we cannot do otherwise.